<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManageRoutes</class>
 <widget class="QMainWindow" name="ManageRoutes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manage Routes</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="routeList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>261</width>
      <height>511</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="remove">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>261</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QComboBox" name="truckSelection">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>11</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Select Truck</string>
    </property>
   </widget>
   <widget class="QListWidget" name="locationList">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>60</y>
      <width>221</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>300</y>
      <width>221</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Route number:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Number of locations:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Total number of items:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Truck capacity:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Current capacity (after loading items):</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>300</y>
      <width>221</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="routeNumber">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="locationCount">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="itemCount">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="truckCapacity">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="currentCapacity">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QListWidget" name="selectedLocations">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>60</y>
      <width>221</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="removeFromSelected">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>260</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="addRouteNumber">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Route Number</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addToList">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>530</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Add To List</string>
    </property>
   </widget>
   <widget class="QPushButton" name="generateRouteInformation">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>530</y>
      <width>171</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate Route Information</string>
    </property>
   </widget>
   <widget class="QPushButton" name="generateLoadingOrder">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>530</y>
      <width>151</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate Loading Order</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
